exercises/practice/rational-numbers/src/test/java/RationalTest.java

```diff
@@ -11,7 +11,7 @@ public class RationalTest {
     private static final double DOUBLE_EQUALITY_TOLERANCE = 1e-15;
 
     private void assertDoublesEqual(double x, double y) {
-        assertThat(x).isEqualTo(y, within(DOUBLE_EQUALITY_TOLERANCE));
+        assertThat(x, closeTo(y, within (DOUBLE_EQUALITY_TOLERANCE));
```

Comment:
Looking at the [AssertJ examples](https://github.com/assertj/assertj-examples/blob/main/assertions-examples/src/test/java/org/assertj/examples/NumberAssertionsExamples.java), I was wondering if this is what you meant?

```suggestion
        assertThat(x).isCloseTo(y, within(DOUBLE_EQUALITY_TOLERANCE));
```

Otherwise this doesn't [compile](https://github.com/exercism/java/actions/runs/12528744783/job/34977757650?pr=2886#step:4:966):

```
/home/runner/work/java/java/exercises/practice/rational-numbers/src/test/java/RationalTest.java:14: error: ')' expected
> Task :practice:rational-numbers:compileStarterSourceJava
        assertThat(x, closeTo(y, within (DOUBLE_EQUALITY_TOLERANCE));
```